pkg/kubelet/dockershim/docker_container.go

```diff
@@ -498,7 +498,7 @@ func (ds *dockerService) performPlatformSpecificContainerCleanupAndLogErrors(con
 
 	errors := ds.performPlatformSpecificContainerCleanup(cleanupInfo)
 	for _, err := range errors {
-		klog.Warningf("error when cleaning up after container %q: %v", containerNameOrID, err)
+		klog.InfoS("Error when cleaning up after container", "container", containerNameOrID, "err", err)
```

Comment:
Needed to think about this more. I would prefer to avoid using `container` for different types of values. Logging `containerNameOrID` without detecting what case it is `ID` or `Name` makes it already not useful for log correlation (looking all logs with same `containerName` field). Best case we would rewrite code to just use one of them, but for now let's just make name explicit, so it will not be wrongly interpreted.
```suggestion
		klog.InfoS("Error when cleaning up after container", "containerNameOrID", containerNameOrID, "err", err)
```